### PR TITLE
Adding an FAQ section

### PR DESCRIPTION
**Pull request by toutantic**
We need a space where we will answer all the questions from the people who wants to develop modules for PrestaShop.

Here are two questions to start with.

### CODE REVIEW

**Comment by matks** [comment]
Probably related to https://github.com/PrestaShop/PrestaShop/issues/14005

File: src/content/1.7/faq/shippingFaq.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- 1.7/faq/
+---
+
+# Shipping FAQ
+
+- [Shipping informations](#shipping-informations)
+- [How do I add a service fee to an order ?](#how-do-i-add-a-service-fee-to-an-order-?)
+
+
+## Shipping informations
```
Comment (eternoendless) [inline src/content/1.7/faq/shippingFaq.md]:
```suggestion
## Shipping information
```

File: src/content/1.7/faq/shippingFaq.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: Shipping FAQ
+alias:
+- 1.7/faq/
+---
+
+# Shipping FAQ
+
+- [Shipping informations](#shipping-informations)
```
Comment (eternoendless) [inline src/content/1.7/faq/shippingFaq.md]:
```suggestion
- [Shipping information](#shipping-information)
```

File: src/content/1.7/faq/shippingFaq.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+## Shipping informations
+Where and how should I read and write the following informations to be sure that my module is compatible with every shipping modules:
+- Tracking number
+- Carrier name
+- Estimated delivery date
+- Real delivery date
+
+TODO:write answer
+
+## How do I add a service fee to an order ?
```
Comment (eternoendless) [inline src/content/1.7/faq/shippingFaq.md]:
```suggestion
## How do I add a service fee to an order? 
```

File: src/content/1.7/faq/shippingFaq.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+title: Shipping FAQ
+alias:
+- 1.7/faq/
+---
+
+# Shipping FAQ
+
+- [Shipping informations](#shipping-informations)
+- [How do I add a service fee to an order ?](#how-do-i-add-a-service-fee-to-an-order-?)
```
Comment (eternoendless) [inline src/content/1.7/faq/shippingFaq.md]:
```suggestion
- [How do I add a service fee to an order?](#how-do-i-add-a-service-fee-to-an-order?)
```

**Pull request merged**